clang_delta/EmptyStructToInt.cpp

```diff
@@ -177,6 +182,49 @@ bool EmptyStructToIntRewriteVisitor::VisitElaboratedTypeLoc(
   return true;
 }
 
+bool EmptyStructToIntRewriteVisitor::VisitRecordDecl(RecordDecl *RD)
+{
+  //if (ConsumerInstance->isSpecialRecordDecl(RD))
+  //  return true;
+
+  // Skip forward declarations
+  const RecordDecl *RDDef = RD->getDefinition();
+  if (!RDDef)
+    return true;
+
+  // Skip the struct which will be deleted
+  // It is already in the list
+  if(RD == ConsumerInstance->TheRecordDecl)
+  {
```

Comment:
Could you change the style to

    if (xxx) {
      stmt;
    }

or 

    if (xxx)
      stmt;